src/operator/contrib/transformer.cc

```diff
@@ -122,6 +122,531 @@ static bool InterleavedMatMulEncDecValAttShape(const NodeAttrs& attrs,
   return true;
 }
 
+void strided_batch_sgemm(bool transA, bool transB,
```

Comment:
Is it better to move this function to the file with linalg prefix?

Reply:
I don't expect this function will be widely used except these mha operators. Also linalg* functions need high level abstraction but I'm not sure how to wrap the pointers a/b/c here into Tensor while still keeping the m/n/k and strides arguments, and how to implement the gpu part.